packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,727 @@
+---
+title: Authenticators Administration
+category: other
+---
+
+# Authenticators Administration API
+
+<ApiLifecycle access="beta" />
+
+The Authenticators Administration API enables an Org Administrator to configure which Authenticators are available to end users to be used for signing into applications.
+
+End users would be required to use one or more Authenticators depending on the security requirements of the application sign-on policy.
+
+The Okta Identity Engine currently supports Authenticators for the following factors:
+
+**Knowledge-based:**
+* Password
+* Security Question
+
+**Possession-based:**
+* Phone (SMS, Voice Call)
+* Email
+* WebAuthn
+
+<!--
+## Get started
+
+Explore the Authenticators Administration API: [![Run in Postman](https://run.pstmn.io/button.svg)](%linkhere%). % How to generate this link: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/227648499/How+to+publish+Postman+collections %
```

Comment:
Is this visible? We don't have a postman collection right now.

Reply:
It's commented out.

Reply:
update: not visible